tests/bootstrap.php

```diff
@@ -130,6 +130,7 @@
 }
 
 ConnectionManager::setConfig('test', ['url' => getenv('DB_DSN')]);
+ConnectionManager::setConfig('test_alt', ['url' => getenv('DB_DSN')]);
```

Comment:
Again I don't care to test the alternate connection, thats up  to the user IMO. We have the exception you've added in place to let them know if they configured the wrong alias.

Reply:
Yep I will remove this